src/tools/crawler/metrics.rs

```diff
@@ -61,6 +65,56 @@ pub fn new_network_summary(crawler: &Crawler, graph: &Graph<SocketAddr>) -> Netw
         }
     }
 
+    let mut node_network_types = Vec::with_capacity(num_good_nodes);
+    for node in &good_nodes {
+        let mut port_matches = false;
+        let mut agent_matches = false;
+        let mut height_matches = false;
+        let mut blocker = false;
+
+        if node.port() == ZCASH_P2P_PORT {
+            port_matches = true;
+        }
+
+        let agent = if let Some(agent) = &nodes[node].user_agent {
+            agent.0.clone()
+        } else {
+            "".to_string()
+        };
+        let zcash_re = Regex::new(r"^/MagicBean:(\d)\.(\d)\.(\d)/$").unwrap();
+        let zebra_re = Regex::new(r"^/Zebra:(\d)\.(\d)\.(\d)").unwrap();
```

Comment:
this provides more context below
```suggestion
        let zcash_regex = Regex::new(r"^/MagicBean:(\d)\.(\d)\.(\d)/$").unwrap();
        let zebra_regex = Regex::new(r"^/Zebra:(\d)\.(\d)\.(\d)").unwrap();
```

Reply:
Changed.
`re` is generally widely used across Regex library (and samples) to indicate if something is regular expression but changed if that wasn't clear enough.

Reply:
>  re is generally widely used across Regex library (and samples) to indicate if something is regular expression but changed if that wasn't clear enough.

aha! I wasn't aware. you could have kept it, I would get used to it if it's commonly used elsewhere